core/modules/query/TableRef.cc

```diff
@@ -205,9 +205,14 @@ std::string TableRef::sqlFragment() const {
 
 void TableRef::putTemplate(QueryTemplate& qt) const {
     auto aliasMode = qt.getTableAliasMode();
+    bool quoteIdentifiers = qt.quoteIdentifiers();
```

Comment:
For read-only variables which are cached it's usually safer to make them `const` like in:
```
bool const quoteIdentifiers = ...
```
This helps code readers and compilers (optimization).